app/assets/javascripts/application.js

```diff
@@ -34,3 +36,53 @@ function scrollTo($el) {
 $(function () {
   $('[data-toggle="tooltip"]').tooltip();
 })
+
+$(function () {
+  addCustomerSelect2('.customer-select2');
```

Comment:
Seems like we can send multiple fields to `function addCustomerSelect2(fields)` hence do it in one line